sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
Are there test cases for this getIdentifier that I missed?

Reply:
Sorry for missing that. will add it.